care/facility/models/json_schema/daily_round.py

```diff
@@ -131,7 +185,23 @@
         {
             "type": "object",
             "properties": {
-                "procedure": {"type": "string"},
+                "procedure": {
+                    "enum": [
+                        "personal_hygiene",
+                        "positioning",
+                        "suctioning",
+                        "ryles_tube_care",
+                        "iv_sitecare",
+                        "nubulisation",
+                        "dressing",
+                        "dvt_pump_stocking",
+                        "restrain",
+                        "chest_tube_care",
+                        "tracheostomy_care",
+                        "stoma_care",
+                        "catheter_care",
```

Comment:
Waiting for updated new set of procedures to be confirmed in 

https://github.com/ohcnetwork/care_fe/pull/8568/files#diff-37705b30476b14631124ce42dd5c3500d43fbede16f0e91f9ae2905e4adc6376R737-R766